scipy/stats/tests/test_stats.py

```diff
@@ -581,6 +581,19 @@ def test_nan_policy(self):
         assert_raises(ValueError, stats.spearmanr, x, x, nan_policy='raise')
         assert_raises(ValueError, stats.spearmanr, x, x, nan_policy='foobar')
 
+    def test_nan_policy_bug_12458(self):
+        np.random.seed(5)
```

Comment:
If you agree w/suggestion proposed beneath then the `np.random.seed` line can be deleted.